docs/FAQ.md

```diff
@@ -108,6 +108,29 @@ You could refer to the above section to try out each of the below prefixes!
 
 **NOTE:** Currently, the above autocompletes ignore a text containing whitespaces if it is specified as a prefix for autocompletion. (e.g., `#test he` wouldn't lead to `#**test here**`) See also [#925](https://github.com/zulip/zulip-terminal/issues/925).
 
+#### Other autocomplete options
+
+All other message composing areas that support autocompletion don't require a prefix, i.e., the prefix for them is an empty string (`''`). Autocompletes here are mostly contextual and specific. See the below snapshots to better distinguish between various compose fields.
+
+|S. No.|Field|Autocompleted text format|
+| :---: | :--- | :---: |
+|1|Stream|`Stream name`|
+|2|Topic|`Topic name`|
+|3|PM recipient|`Name <email>`|
```

Comment:
Given you have two images for Streams and PMs, I'd suggest skipping the text above and this table, and having two sections referring to 1, 2 and 3 in the text in each section. The table doesn't really give much extra information here.

The notes can also fit into the sections.